lib/config/configuration.go

```diff
@@ -2041,6 +2051,23 @@ func applyWindowsDesktopConfig(fc *FileConfig, cfg *servicecfg.Config) error {
 	return nil
 }
 
+func staticHostsWithAddress(hosts []WindowsHost, defaultPort int) ([]servicecfg.WindowsHost, error) {
+	var hostsWithAddress []servicecfg.WindowsHost
+	for _, host := range hosts {
+		addr, err := utils.ParseHostPortAddr(host.Address, defaultPort)
```

Comment:
I disagree.

RDP operates on TCP port 3389, so defaulting to TCP is absolutely correct in this case. It doesn't really matter anyway since we don't use Go to dial the desktop.

Reply:
Ah true, somehow I got it in my head that we mean https here. I still think documenting the default behavior on `ParseHostPortAddr` is ideal.

Reply:
I always declare hosts without scheme (didn't know it was an option to start with) and as Zac mentioned TCP is correct for RDP, I'll leave it as it is